### DISCUSSION TITLE
Tasmota, Domoticz and Sonoff basic

### DISCUSSION THREAD

**Discussion by carminou** [arendst/Tasmota#15778] [Q&A] [answered]
Hello,
new on this topic, I try to make Domoticz and Tasmota flashed Sonoff basic (R1 and R3) switches work together.

The Sonoff switch is running Tasmota v 11.1.0 and respond correctly through the web interface.
Spying the MQTT server, I see messages like:

> stat/tasmota_24ED35/RESULT {"POWER":"OFF"}
> stat/tasmota_24ED35/POWER OFF
> stat/tasmota_24ED35/RESULT {"POWER":"ON"}
> stat/tasmota_24ED35/POWER ON

when I press the Sonoff basic button (and the led and the relay switch)
or periodically:
> tele/tasmota_24ED35/STATE {"Time":"2022-06-09T17:21:37","Uptime":"0T00:05:11","UptimeSec":311,"Heap":27,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":25,"MqttCount":1,"POWER":"ON","Wifi":{"AP":1,"SSId":"Lejardindesfleurs","BSSId":"28:D1:27:81:00:E2","Channel":6,"Mode":"11n","RSSI":100,"Signal":-16,"LinkCount":1,"Downtime":"0T00:00:05"}}

In Domoticz... Nothing.
So in Domoticz I created an MQTT hardware  of type "MQTT Auto Discovery Client Gateway with LAN interface" and with "Auto Discovery Prefix" set to "tasmota". It still doesn't work, but I get messages in Domoticz log:
>  2022-06-09 18:31:22.244 Status: MQTT Broker tasmota: discovered: c1aa449a-d5d6-495f-b131-8d32e3903372/c1aa449a-d5d6-495f-b131-8d32e3903372 (unique_id: c1aa449a-d5d6-495f-b131-8d32e3903372)
> 2022-06-09 18:31:22.344 Error: MQTT Broker tasmota: MQTT_Discovery: Something other then 'config' received on discovery topic! (discovery/B4E62D24ED35/sensors/{"sn":{"Time":"2022-06-09T17:31:22"},"ver":1})!
> 2022-06-09 18:31:22.344 Error: MQTT Broker tasmota: MQTT_Discovery: Invalid/Unhandled data received! (Topic: discovery/B4E62D24ED35/sensors, Message: {"sn":{"Time":"2022-06-09T17:31:22"},"ver":1})

Has anyone already succeeded setting up this kind of configuration? or any help to solve this?
Regards

**Comment by sfromis** [comment]
I'm not using Domoticz, but in my understanding you need to specifically configure Tasmota to publish to MQTT in the very specific format needed by Domoticz, and be using the "Configure Domoticz" item in the Tasmota web UI for setup. And of course first read the docs about the integration....
https://tasmota.github.io/docs/Domoticz/

**Reply by carminou** [reply]
I already read this and it works. I was trying to work with the auto discovery client... Maybe it is not feasible?

**Reply by sfromis** [reply]
In that case, the question is no longer about Tasmota, but about that thing you are trying to make work

**Reply by carminou** [reply]
Answering to myself:
in Tasmota: setoption19 1 to enable homeassistant discovery....

**Comment by carminou** [comment, marked as answer]
Answering to myself:
in Tasmota: setoption19 1 to enable homeassistant discovery....
